lib/common/options.js

```diff
@@ -60,6 +60,25 @@ Options.from = function (_options) {
     : options.rootDir
   options.buildDir = path.resolve(options.rootDir, options.buildDir)
 
+  // Dummy proof buildDir and generate.dir as those folders
+  //  will be completely removed when used
+  if (options.rootDir.startsWith(options.buildDir)) {
+    const errorMessage =
+      'options.buildDir cannot be set lower than or equal to rootDir'
```

Comment:
`options.buildDir cannot be inside or same as rootDir` isn't better?

Reply:
For me inside means on a higher level, not a lower level. So `/path/subpath` is inside `/path` which is the opposite of what we mean. If you think that as well, we could maybe use outside?

Reply:
True! I'm drunk man

Reply:
Hmm, while changing this I am thinking that outside is not specific enough. Eg `/build-path` is outside `/nuxt/my-project` and should normally not be a problem. Its really only paths up the same tree as rootDir